scripts/subreddits_done.txt

```diff
@@ -1,1119 +1,95 @@
-SexyGirlsInJeans
-hotsoccerfans
+WtSSTaDaMiT
+curvy
 Bigtitssmalltits
-WesternHentai
-DickSlips
-NextDoorBoobies
-shavedpussies
-CarmellaRose
-usedpanties
-OralCreampie
-Boxers
-slightlypregnant
-ravergirl
-Katya_Sambuca
-FFSocks
-randomsexiness
-CumOnTheirFaces
-apotheoun
-mentalhealth
-OnlyHotMilfs
-DaniThorne
-PhotoShopBattlesNSFW
-GirlsWashingCars
-BlowjobGifs
-redditamateurvids
-findom
-BellaThorne
-EmmyRossum
-BlacksOnAsians
-dopplebangher
-amateurcumsluts
-brunettes
-Hips
-EllieKemper
-PublicUpskirts
-cutepanties
-TeeDubsBooty
-RachelCook
-redhead
-cumfetish
-AlAnon
-inthenameofkittens
-survivorbeauty
-OldenPorn
-UNBGBBIIVCHIDCTIICBG
-wolves
-ClassicScreenBeauties
-GWCouples
-MissAlice_18
-SorryIAmAChampion
-ChangingRooms
-AimeeTeegarden
-YoungMonroe
-Bondage
-ComplexionExcellence
-manbuns
-bestofcollege
-GoneMildPlus
-handjobs
-Bottomless_Vixens
-bleps
-insanelyhairymen
-cuteguys
-JiggleFuck
-StrongCurves
-PetiteGoneWild
-gfur
-hsp
-ImpresssedByCum
-Ratemeteen
-BubbleButts
-pullingdownthepanties
-masturbation
-PussyFlashing
-underarms
-SauceforNSFWads
-happygirls
-CourtneyTailor
-leannadecker
-LingerieGW
-webcamgirls
-WiggleButts
-Blackdick
-gingerdudes
-Maxisma
-amateuroral
-SolesAndButtholes
-theratio
-NSFW_GIF
-Cougar
-BustyAmateurs
-primes
-gothsluts
-MetArt
-AgedBeauty
-BreastExpansion
-BlowJob
-exotic_oasis
-Sexy
+gonewild
+seethru
+iWantToFuckHer
+diorabaird
+SexyGirlsInBoots
+thinspo
+womenofcolour
+hot_pics
+Ginger
+SexyGirlsInJeans
+BustyPetite
+nsfwoutfits
+malemodels
 Unashamed
-leannecrow
-HighMileageHoles
-bigdickjoy
-BeardPorn
-TheUnderbun
-DemiRoseMawby
-Innie
-storyofafuckering
-Aprons
-NSFW_Tributes
-Footjob
-DSLs
-8chan
-PussyMound
-ankletporn
-amihot
-PCOS
-firstimpression
-softies
-OldSchoolBabes
-overflowingbra
-palebeauties
-AvPD
-Helen_Owen
-realasians
-couplesgonewildplus
-BathtubBeauties
-CelebsInTights
-Cum_Covered_Hentai
-penismeasured
-fffxxx
-LindsayElyse
-bimbofetish
-tubetop
-GoneWildSmiles
-gonewildcouples
-truthfulteenopinions
-Zishy
-asiangirls
-OfficialNYLDY
-LatinasGW
-sizecomparison
-SANIC
-VietnameseHotties
-Hot100
-Daddypics
-LegalTeens
-pantyslide
-LabiaGW
-Malmalloy
-iamatease
-ThickChixxx
-skivvies
-distension
-realgirlfriends
-ModelsGoneMild
-GirlsInSocks
-GirlsWithBareFeet
-ArmsUp
-DirtyPantiesGW
-CelebrityPenis
-preggohentai
-gaysian
-pitchforkemporium
-BABES4PORN
-NSFWCostumes
-petite
-Blueberry
-ForeheadCum
-surpriseddogs
-FitToFat
-leggingsgonewild
-PetiteGirls
-Denise_Milani
-RugsOnly
-assholegonewild
-PickOne
-GillianJacobs
-peachfuzz
-enantiodromia_vault
-AnalOrgasms
-TallGoneWild
-asaakira
-cuckoldstories
+sexyfrex
+OnOff
+BBW
+BonerMaterial
+Porn_imgur
+Porn_beauty
+GroupOfNudeGirls
+SuicideGirls
+ladybonersgw
+celebNSFW
+Playboy
+womenofcolor
 nsfw
-Naughtymacy
-milf
-4kpornmovies
-JapaneseUncensored
-PornMusicVids
-AsianCumsluts
+hairbra
+prettygirls
+Horny_Porn_Pics
+jockstraps
+PhotoShopBattlesNSFW
+BABES4PORN
+malespandex
+slingbikini
+jockwatch
+mangonewild
 BHMGoneWild
```

Comment:
this file is just a cache that gets filled as you run the scraper script longer. when we run it in dev we'll have more images.